near-state-indexer/src/utils.rs

```diff
@@ -59,16 +59,16 @@ pub(crate) async fn fetch_status(
         .await??)
 }
 
-pub(crate) async fn publish_streamer_message(
-    topic: &str,
+pub(crate) async fn update_block_streamer_message(
+    block_type: &str,
```

Comment:
I really don't like the approach of using error-prone `&str` in the world of Rust-lang where we have fancy `enum`s that can prevent us from